README.md

```diff
@@ -300,6 +301,12 @@ We highly recommend installing a [miniconda](https://docs.conda.io/en/latest/min
 
     chances are your libGL is located at a non-standard location. See e.g. [this issue](https://askubuntu.com/questions/541343/problems-with-libgl-fbconfigs-swrast-through-each-update).
 
+## Documentation
+
+Habitat-Sim documentation is available [here](https://aihabitat.org/docs/habitat-sim/index.html). 
+
+For example, see [this page](https://aihabitat.org/docs/habitat-sim/lighting-setups.html) for adding new objects to existing scenes and relighting the scene & objects. See [this page](https://aihabitat.org/docs/habitat-sim/image-extractor.html) for extracting images from scenes loaded in Habitat-Sim. 
```

Comment:
(Sorry if this sounds too nitpicky, but I can't just ignore it.) Could this be rephrased so that the link text alone describes what's under the link? Having just `here`, `click here` or `this page` is an [accessibility antipattern](https://ui-patterns.com/blog/User-Interface-AntiPatterns). Suggested edit:

```suggestion
Browse the online [Habitat-Sim documentation](https://aihabitat.org/docs/habitat-sim/index.html). 

To get you started, see the [Lighting Setup tutorial](https://aihabitat.org/docs/habitat-sim/lighting-setups.html) for adding new objects to existing scenes and relighting the scene & objects. The [Image Extractor tutorial](https://aihabitat.org/docs/habitat-sim/image-extractor.html) shows how to get images from scenes loaded in Habitat-Sim. 
```